test/toniq/job_process_test.exs

```diff
@@ -25,20 +25,22 @@ defmodule Toniq.JobProcessTest do
     assert Toniq.JobProcess.run(job) == {:job_was_successful, job}
   end
 
-  test "a job that raises an error returns {:job_has_failed, job, error}" do
+  test "a job that raises an error returns {:job_has_failed, job, error, stack}" do
     job = %{worker: TestErrorWorker, arguments: [data: 10]}
-    assert Toniq.JobProcess.run(job) == {:job_has_failed, job, %RuntimeError{message: "fail"}}
+    result = Toniq.JobProcess.run(job)
+    assert {:job_has_failed, ^job, %RuntimeError{message: "fail"}, _} = result
+    assert {_, _, _, [{TestErrorWorker, :perform, 1, _}|_]} = result
```

Comment:
good point, @joakimk, just ammended the commit, making the stack trace check more explicit 🙂